eman2/viewers/views_tkinter_tree.py

```diff
@@ -0,0 +1,92 @@
+# **************************************************************************
+# *
+# * Authors:     David Herreros (dherreros@cnb.csic.es)
+# *
+# * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+import glob
+import os
+import threading
+
+from pyworkflow import utils as pwutils
+from pyworkflow.gui.dialog import ToolbarListDialog
+from pyworkflow.utils.path import moveFile, cleanPath, copyFile
+from pyworkflow.utils.process import runJob
+
+import eman2
+
+
+class EmanDialog(ToolbarListDialog):
+    """
+    This class extend from ListDialog to allow calling
+    an Eman subprocess from a list of Tomograms.
+    """
+
+    def __init__(self, parent, path, **kwargs):
+        self.path = path
+        self.provider = kwargs.get("provider", None)
+        self.inMemory = kwargs.get("inMemory", None)
+        ToolbarListDialog.__init__(self, parent,
+                                   "Tomogram List",
+                                    allowsEmptySelection=False,
+                                    itemDoubleClick=self.doubleClickOnTomogram,
+                                    **kwargs)
+
+    def refresh_gui(self):
+        if self.proc.isAlive():
```

Comment:
isAlive() is deprecated, use is_alive() instead

Reply:
Fixed in https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/65849fe68edd7b8b9b4430ba83bb1b9c572d7b84